### ISSUE TITLE
Stats polish: Posting activity UI

### ISSUE THREAD

**Issue by jeffgolenski** [Automattic/wp-calypso#70609]
# Development

### Must have:
- [x] Restyle container and header.
  - [x] Remove card borders.
  - [x] Enlarge the title and change font to match the headers on the Traffic page.

- [x] Restyle "day" squares within each month. 
  - [x] Add 2px border-radius to each square.
  - [x] Increase size of each day square to be 12x12px. Padding between each should be 4px.

- [x] Restyle the chart legend ("Fewer ■■■■■ More Posts").
  - [x] Should be centered and enlarged.
  - [x] Remove padding between squares in this area.
  - [x] Add 5px rounded corners.

- [x] Orient each month square horizontally instead of vertically. [See Filipe's comment](https://github.com/Automattic/wp-calypso/issues/70609#issuecomment-1333620075).

- [x] Make the interface more responsive.
  - [x] Show the posting activity section in a mobile viewport.
  - [x] Reduce the number of months shown depending on the viewport width.

### Nice to have:

- [ ] Improve styling when hovering over a day square.
  - [x] Display the date, day of the week, and amount of posts published in the hover tooltip.
  - [x] Increase the size of the hovered "day" square to 14x14 px, if possible.
  - [ ] Drop all other day squares' opacity to 50% when a day square is being hovered.

- [x] ~~Add arrow navigation so users can scan back past 12 months.~~ (This is no longer a valid task -@jsnmoon)

Must-have items must be completed before closing this issue. Please spin off any unfinished tasks into separate issues.

# Design

### Details

Just making note of a few refinements to make on the insights graphs. Nothing major, just some visual polish. Breaking down each section for easier consumption

### **Posting activity**

**Current**
![Screen Shot 2022-11-30 at 5 22 49 PM](https://user-images.githubusercontent.com/214813/204921058-2a41e69d-d150-403d-aa73-884ff73e83bf.png)

**After:**
<img width="1792" alt="Post activity" src="https://user-images.githubusercontent.com/214813/204922717-c9f1e167-1282-40f4-8ac0-8efde24e1d45.png">

![Screen Shot 2022-11-30 at 5 40 00 PM](https://user-images.githubusercontent.com/214813/204923706-7b259bbc-057d-411e-a8f6-b394002ce125.png)

In context:
<img width="2963" alt="Insights i2 - Dotcom" src="https://user-images.githubusercontent.com/214813/204922731-f8260b99-d467-44c3-8e76-e50277d0c446.png">

### Checklist

- [ ] Overall page: When ready, the layout of most areas should be broken out of the bordered cards, just like we've done with the Traffic page!

<details><summary>Migrated to Development section</summary>

- [x] ~Add rounded corners to each little "day" square within each month. 2px radius on each square.~
- [x] ~Increase size of each day square to be 12x12px. Padding between each should be 4px.~
- [x] ~Enlarge title / change font to match headlines on Traffic page~
- [x] ~Fewer / More post legend should be centered and enlarged. Please remove padding between squares in this area and add 5px rounded corners as shown in the mock!~

On hover:
- [x] ~Tooltip should display with the date, day, and amount of posts published~
- [x] ~"day" square should increase in size to 14x14 px (if possible!) and all other day squares should drop to 50% opacity, so it's clear which one is hovered (see close up above)~

Bigger tasks:
- [x] ~Build arrow navigation so users can scan back past 12 months~

</details>
cc @keoshi

**Comment by keoshi** [comment]
Worth mentioning the squares for each day are now laid out horizontally as you'd expect from any calendar view instead of vertically. Here's a quick before and after:

![image](https://user-images.githubusercontent.com/390760/205041143-b2b64112-9ae1-45ca-b79f-bc0bda6962d3.png)

**Comment by wilmersondasilva** [comment]
An update on this one: The [PR 71580](https://github.com/Automattic/wp-calypso/pull/71580) containing all the **must-have** items and almost all the **nice to have** items was deployed to production yesterday. Today, a [follow-up PR](https://github.com/Automattic/wp-calypso/pull/71676) was also deployed. 

>  Add arrow navigation so users can scan back past 12 months.

It was [decided](pejTkB-ag-p2#comment-163) that this is no longer needed. So, the [PR](https://github.com/Automattic/wp-calypso/pull/71603) containing it was closed.

Some of the decisions related to this task were discussed in this [P2 post](pejTkB-ag-p2).

I'm moving this one to "Done"

**Comment by jsnmoon** [comment]
@wilmersondasilva, could you create a follow-up issue for the last remaining nice-to-have task (dropping unselected square opacity) and close this issue? Thanks!

EDIT: I also realized we need to move the posting activity interface above the all-time insights heatmap like so:

<img width="711" alt="image" src="https://user-images.githubusercontent.com/4044428/211107548-419d6997-bce2-41d2-8eaf-5a71ffec305d.png">

Could you spin up another issue for tracking this as well?
